distribution/receiver/routes/shutdown-route.xml

```diff
@@ -2,8 +2,14 @@
     <route id="shutdown-route">
         <from uri="direct:shutdown-route" />
 
-        <!-- TODO Send a notification to the configured emails -->
-
+      	<doTry>
+        	<toD uri="direct:send-notification-email-route" />
```

Comment:
Call the processor directly from here and pass it the receiverId from here on the exchange